src/main/java/business/models/LabRequestRepository.java

```diff
@@ -27,6 +28,12 @@
 
     List<LabRequest> findAllByLabIn(Collection<Lab> labs, Sort sort);
 
+    /* Since sent_return_email was added later, not all labrequests have it set. Using COALESE means all items that are
+     * null will get evaluated as being FALSE */
+    @Query(value  = "SELECT * FROM lab_request lr WHERE lr.return_date < to_timestamp(:now, 'YYYY-MM-DD HH24:MI:SS.US') "+
```

Comment:
Or make the field not nullable and update the database to set all entries to false if not set. Why did you choose this options (which is a valid one)?